packages/components/src/Button/Button.css

```diff
@@ -1,16 +1,15 @@
 :root {
-  --button--color-variation: var(--color-interactive);
-  --button--color-variation--hover: var(--color-interactive--hover);
+  --public--button--color-variation: var(--color-interactive);
+  --public--button--color-variation--hover: var(--color-interactive--hover);
```

Comment:
I think we're missing a step here if I'm not mistaken.

if we only changing the variable name from `--button--color-variation` to `--public--button--color-variation` but we don't update the places that are consuming it via `var(--button--color-variation)` then the public version isn't being used.

if I throw a random button into a page somewhere I see this

![image](https://github.com/GetJobber/atlantis/assets/11843215/0a1152c8-826f-4c5a-a7e2-53f871fcafe5)

where it's getting those variables defined by

```
/* Variation */

.qkFQwI0PKfw- {
  --button--color-variation: var(--color-interactive);
  --button--color-variation--hover: var(--color-interactive--hover);
}
```

which is the unique-ified version of this rule

```
/* Variation */

.work {
  --button--color-variation: var(--color-interactive);
  --button--color-variation--hover: var(--color-interactive--hover);
}
```

Reply:
Oh of course, very good catch